src/main/launcher/property.c

```diff
@@ -294,11 +294,24 @@ static void boot_property_avs_rewind(uint32_t context)
 
 struct property *boot_property_load_avs(const char *filename)
 {
+    uint16_t mode;
     avs_desc desc;
     struct property *prop;
 
-    // TODO changing mode from read mode to 0 makes DDRX3 reading the ea3-config work, otherwise it fails querying it
-    desc = avs_fs_open(filename, 0, AVS_FILE_FLAG_SHARE_READ);
+    // Apparently, the mode enum flags have changed, but this is currently unverified to apply
```

Comment:
read 0x00, write 0x01, readwrite 0x03

Reply:
Awesome, thanks for that. Pushed a commit to address that.